### ISSUE TITLE
It's unclear what "Relevance" means in the search options

### ISSUE THREAD

**Issue by trang** [Tatoeba/tatoeba2#1926]
Reported by [Aiji on the Wall](https://tatoeba.org/eng/wall/show_message/32234#message_32234):

> UX-ly speaking (yes, I'm creative word-maker), I think that would be a good idea to add a question-mark pop-up on the right of the "Order" select box of the advanced search to explain what this mysterious "Relevance" is about. Right now, that's pretty much 100% unclear.

**Comment by alanfgh** [comment]
In the meantime, I will describe it on the wiki. However, first I need to find out what it means. My guess is that "Relevance" is the order that the Manticore search engine uses by default (as modified by our configuration) to display results. It favors sentences with the fewest words, but the overall algorithm is more complicated than that. @jiru , is it possible to describe the algorithm, or point me to a place (some combination of Manticore documentation and our own configuration) where I can figure it out?

**Comment by AndiPersti** [comment]
@alanfgh: The relevant documentation page for Manticore is https://docs.manticoresearch.com/latest/html/searching/search_results_ranking.html#search-results-ranking.
The ranking formula we use for "Relevance" is defined at https://github.com/Tatoeba/tatoeba2/blob/3e6de6c72e2dc19fef146792a131e327c022cd81/src/Model/Table/SentencesTable.php#L1325

jiru's commit that added that formula is 47b56f1151bde721843c8adfc2545d2f91ab881f

**Comment by alanfgh** [comment]
I see. So "Relevance" favors shorter sentences, but modifies that preference by favoring sentences that:
(1) have a long common subsequence ("lcs") in common with the query, and 
(2) match all query keywords, and in the same order as in the query
Many thanks, @AndiPersti !

**Comment by jiru** [comment]
See also the explanation I left in commit 47b56f1151bde721843c8adfc2545d2f91ab881f.

**Comment by alanfgh** [comment]
Got it. Thanks, @jiru !

**Comment by agrodet** [comment]
I was wondering what would be a good way to display this information. I made a quick prototype of what, I think, is a possible option. Actually I wanted to display what's below but I couldn't find how to display a multiline tooltip! (The first two results I found didn't work and I didn't want to spend much time on a prototype :D If someone knows how to do it, I'm all open)
```
The relevance option prioritizes, in order:
1. Sentences that contains an exact match.
2. Sentences having the longest common subsequence.
3. Sentences having the least number of words.
```

![image](https://user-images.githubusercontent.com/42261966/74643322-e0c39300-51b7-11ea-8e3e-78a2f377989a.png)

Any thoughts?

**Comment by AndiPersti** [comment]
Do you use `md-tooltip`? You would need to [overwrite the hard coded `height` property.](https://stackoverflow.com/questions/33293599/how-to-add-line-breaks-within-tooltip-in-angular-material-design)

**Comment by agrodet** [comment]
Yeah, I tried that, but it didn't work. I'll check more seriously if we go with this option.

**Comment by AndiPersti** [comment]
Well it works for me with Firefox:

![tooltip](https://user-images.githubusercontent.com/6566321/74671452-7f95c280-51ab-11ea-805e-3c75bbe35a13.png)

In the template file:

    <md-icon>
        help_outline
        <md-tooltip class="multiline-tooltip">
            <?= __('The relevance option prioritizes, in order:') ?></br>
            <?= __('1. Sentences that contains an exact match.') ?></br>
            <?= __('2. Sentences having the longest common subsequence.') ?></br>
            <?= __('3. Sentences having the least number of words.') ?>
        </md-tooltip>
    </md-icon>

In the css file:

    .multiline-tooltip .md-content {
        height: auto;
    }

**Comment by jiru** [comment]
What about also renaming "relevance" into "closest match" that @alanfgh [suggested elsewhere](https://github.com/Tatoeba/tatoeba2/pull/2108#issuecomment-583938660)?
